apps/system/test/marionette/statusbar_icon_color_test.js

```diff
@@ -77,9 +77,10 @@ marionette('Statusbar colors', function() {
   test('statusbar icons keep color after add homescreen', function() {
     waitVisible();
     helper.unlockScreen(client);
+    waitForDarkColor();
     var url = server.url('sample.html');
     bookmark.openAndSave(url);
-    waitForLightColor();
+    waitForDarkColor();
```

Comment:
Hmmm, why are we changing these to dark? Is it because of https://bugzilla.mozilla.org/show_bug.cgi?id=1168915?